projects/public-search/src/app/patron-profile/patron-profile.component.ts

```diff
@@ -110,6 +113,10 @@ export class PatronProfileComponent implements OnInit, OnDestroy {
 
   /** OnInit hook */
   ngOnInit(): void {
+    this.feeTotal = 0;
+    this._subscription.add(
+      this._patronProfileService.loanFeesEvent$.subscribe((fees: number) => this.loansFeesTotal += fees)
```

Comment:
Javascript and floating number are problematic... you need to round the result with 2 decimals. See this simple exemple :
```javascript
let total = 0.01;
console.log(total);
for (let idx=0; idx<10; idx++) {
 total += 0.01;
 console.log(total);
}
```
result
```
0.01
0.02
0.03
0.04
0.05
0.060000000000000005
0.07
0.08
0.09
0.09999999999999999
0.10999999999999999
```